lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -728,8 +734,20 @@ protected function prepare_status_response( $comment_approved ) {
 	protected function prepare_item_for_database( $request ) {
 		$prepared_comment = array();
 
-		if ( isset( $request['content'] ) ) {
-			$prepared_comment['comment_content'] = $request['content'];
+		if ( isset( $request['content'] ) && is_string( $request['content'] ) ) {
```

Comment:
See similar logic in the Posts Controller: https://github.com/WP-API/WP-API/blob/develop/lib/endpoints/class-wp-rest-posts-controller.php#L782